### ISSUE TITLE
Translate May 15 covid report into Dutch

### ISSUE THREAD

**Issue by cassiawag** [nextstrain/translations#182]
Happy weekend @nextstrain/dutch,

The situation report from 2020-05-15 is ready to be translated to Dutch. This week's report is short and focuses on mutations. It's targeted more at the public than some of our previous reports and is intended to answer some of the many questions we've been receiving on the subject. I hope it will be useful for Dutch speakers.

Sadly, this will be our last weekly Situation Report for awhile although we'll issue periodic sit-reps as needed. Thank you, thank you, thank you for translating these reports. It's been an amazing collaborative effort to be involved in. I'll miss working with you on them.  I've personally derived a lot of global solidarity from our work together. Although we will all probably enjoy having more time on the weekend!

Branch to translate on: `sit-rep_nl`

File to translate: https://github.com/nextstrain/translations/blob/sit-rep_nl/narratives/translations_ncov_sit-rep_nl_2020-05-15.md You can translate directly in this file, but please remember to push often as other people may be working on the translation!

Link to preview: https://nextstrain.org/community/narratives/nextstrain/translations@sit-rep_nl/ncov/sit-rep/nl/2020-05-15

Twitter post to translate:
“Our latest situation report focuses on viral mutations: why they occur, when they matter, and how they can be useful in tracking outbreaks. The report is now available in Dutch.

Since repeated patterns of viral introduction and spread across the world have become clear, we'll be pausing weekly situation reports for a while. We will publish periodic situation reports as the data allows.

Thank you to @, @, @ for their rapid translation of this report. Thank you to @, @ for their work on previous translations as well."

Regards,
Cassia

**Logistics**
Please update the translations board as you work on the translation: https://github.com/orgs/nextstrain/projects/7

Please assign yourself to the translation if you’re working on it this week.

When a translation is ready for publication -- at minimum meaning two people have worked on it and both agree that it’s ready -- please create a pull request to merge to `master` and tag me and the issue in it!

Discussion board for your team is here: https://github.com/orgs/nextstrain/teams/dutch

**Standard rules for translation:**
Note that the file to translate is in Markdown. In general, it is important _not_ to change any formatting or links. (Although it is okay to update Wikipedia or CDC links to the appropriate language version.) Comments to help with translation are noted in comment symbols like `<! -- Comment -->`.

Please add your name under `translators:` with new line, tab, and dash, formatted just like authors are. Please also add a link to your professional page (can use Twitter) under `translatorLinks:`.

Note: You may know of other great resources available in Dutch; please don’t add them to the situation report. We need to keep content the same across all translations, so we can be confident in the accuracy of what we publish. Thanks for understanding.

At minimum, I require two sets of eyes on every translation; one of those people should have a background in phylogenetics. Although if you have more, that’s fine too.

Feel free to use a live, online editor like [hackmd](https://hackmd.io/) (sign in with GitHub) to translate together.

**Comment by JiscaH** [comment]
@roanvanscheppingen  could you take this one please?

**Comment by roanvanscheppingen** [comment]
haha it was already on my to do list. I'll get my groceries, inhale my coffee en start working on it!

**Comment by roanvanscheppingen** [comment]
Most of it done. Proofreading would be nice. I have 1 thing left in caps between brackets, because I was not sure of it. Was difficult to find the right Dutch words carrying the right nuance this time! 

Was a pleasure helping you, Nextstrain team <3

**Comment by roanvanscheppingen** [comment]
@nextstrain/dutch  someone able to proofread?

**Comment by cassiawag** [comment]
@JSHuisman or @rvosa, are either of you available to review? Last sit-rep for a while!

**Comment by JiscaH** [comment]
I've gone over @roanvanscheppingen 's translation with a fine-tooth comb, and fixed a bunch of typo's + some awkward sentence structures. Throughout I've changed viral 'streng' to 'stam' - I got way more relevant google hits for the latter translation. 

It would be great if @JSHuisman or @rvosa could give it a final read through, but if not I think it's OK to go live. Up to @cassiawag to make the hard choices ;)

I like the new nextstrain main page btw! Makes it easier to look back on how it all began.
